firestore/cloud-async-client/snippets_test.py

```diff
@@ -0,0 +1,308 @@
+# Copyright 2017, Google, Inc.
```

Comment:
I think there may be enough change to rev this?

Reply:
+1, I think there are enough differences to bump this to 2020

Reply:
Resolved!